resources/css/style.css

```diff
@@ -45,9 +45,22 @@
 .title-get-involved {
   font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
```

Comment:
hey, your shortlog isnt according to our standards. It should be "Improve" instead of "Improved"..see the imperative form :) ?

also, enclose "Get Involved" in double backticks: ``